packages/gen-ai/bff/internal/helpers/logging.go

```diff
@@ -39,6 +39,7 @@ var sensitiveHeaders = []string{
 	"X-Auth-Request-Groups",
 	"X-Envoy-Peer-Metadata",
 	"X-Envoy-Peer-Metadata-Id",
+	"X-Mcp-Bearer",
```

Comment:
@Schimuneck Does it make a difference if it's X-Mcp-Bearer or X-MCP-Bearer? I can see it's X-MCP-Bearer in the codebase.

Reply:
/lgtm /approve based on @Lucifergene comment. Ty!